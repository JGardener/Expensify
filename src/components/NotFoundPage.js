import React from 'react'
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
<p>404! - This page wasn't found! <Link to="/">Go home</Link></p>
);

export default NotFoundPage;