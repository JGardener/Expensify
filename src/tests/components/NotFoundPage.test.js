import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import expectExport from 'expect';

test("Should render the NotFoundPage component", () => {
    const wrapper = shallow(<NotFoundPage />);
    expectExport(wrapper).toMatchSnapshot();
} )