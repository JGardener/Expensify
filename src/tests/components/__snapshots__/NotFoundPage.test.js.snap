// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the NotFoundPage component 1`] = `
<p>
  404! - This page wasn't found! 
  <[object Object]
    to="/"
  >
    Go home
  </[object Object]>
</p>
`;
